### PR TITLE
Allow different units for duration input

### PR DESCRIPTION
**Pull request by tacigar**
I tried to implement 2nd idea of #2208 .

μs
---
e2fd44d074cb28b784c46bc6fcab9982

ms
---
c2ed6da055f891b75dfd760297cbb1fb

s
---
c85c551de9b1969da64cf54ef14e051f

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
cool! looks good


nit maybe the text above no longer needs to say us? maybe it does..

**Comment by tacigar** [comment]
Yes.
I considered microseconds as the default unit and treat durations as microsecond if we do not specify a unit.
But currently microsecond units can not be explicitly specified.
I will change this...

**Comment by tacigar** [comment]
Support μs units.

cb87ac65f3c3d9f1203cc4f974fbe3b2

**Comment by drolando** [comment]
Lgtm!

Can you also add a tip in the textbox showing the possible options?

**Comment by codefromthecrypt** [comment]
screen shot 2018-11-19 at 10 14 18 am
@drolando how about this? there's not much room..

**Comment by codefromthecrypt** [comment]
just added a commit

**Comment by tacigar** [comment]
Great!
Thank you~ 👍

**Comment by codefromthecrypt** [comment]
@drolando if you want change after no problem. hopefully we don't have to describe traces that took minutes or hours too much :D

File: zipkin-ui/js/component_data/default.js [hunk 1]
```diff
@@ -6,8 +6,31 @@ import {traceSummary, traceSummariesToMustache} from '../component_ui/traceSumma
 import {mergeV2ById} from '../spanCleaner';
 import {SPAN_V1} from '../spanConverter';
 
+export function convertDurationToMicrosecond(duration) {
+  const match = duration.match(/^(\d+)(μs|ms|s)$/i);
+  if (match) {
+    const unit = match[2];
+    switch (unit) {
+      case 'μs':
```
Comment (shakuzen) [inline zipkin-ui/js/component_data/default.js:14]: Not that I imagine I'll be using microsecond units, as not having to use that was the point of the change, but I think many people don't know how to easily type `μ` on their keyboard. What if we also accept `us` for microseconds?

Comment (codefromthecrypt) [reply zipkin-ui/js/component_data/default.js:14]: good point.. I think lack of suffix should be μs. Is it not already the case? good suggestion on us as well.

Comment (tacigar) [reply zipkin-ui/js/component_data/default.js:14]: yes, lack of suffix will be μs now.
but I think `us` is also good.
I think we should support both of them.
I'll make another PR.

**Pull request merged**